Theory/ELF.md

```diff
@@ -131,26 +135,26 @@ ELF Header:
   Section header string table index: 70
 ```
 
-Here we can see that `vmlinux` is 64-bit executable file.
+Here we can see that `vmlinux` is a 64-bit executable file.
 
-We can read from the [Documentation/x86/x86_64/mm.txt](https://github.com/torvalds/linux/blob/master/Documentation/x86/x86_64/mm.txt):
+We can read from the [Documentation/x86/x86_64/mm.txt](https://github.com/torvalds/linux/blob/master/Documentation/x86/x86_64/mm.txt#L19):
 
 ```
 ffffffff80000000 - ffffffffa0000000 (=512 MB)  kernel text mapping, from phys 0
 ```
 
-So we can find it in the `vmlinux` with:
+We can then look this address up in the `vmlinux` ELF object with:
 
 ```
-$ readelf -s vmlinux | grep ffffffff81000000
+readelf -s vmlinux | grep ffffffff81000000
```

Comment:
Please, return `$` symbol to not confuse code with output of the output of the cli